src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -139,15 +141,27 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
     /**
      * Filter DataAccessRequestManage objects on user.
      *
-     * @param authUser Filter on what DACs the auth user has access to.
+     * @param user Filter on what DARs the user has access to.
      * @return List of DataAccessRequestManage objects
      */
-    public List<DataAccessRequestManage> describeDataAccessRequestManageV2(AuthUser authUser) {
+    public List<DataAccessRequestManage> describeDataAccessRequestManageV2(User user, Optional<String> roleName) {
```

Comment:
Optional is fine as an argument for the resource class, but the service class should take a plain `String` ([see link](http://dolszewski.com/java/java-8-optional-use-cases/)) and then handle the null case specifically.